daemon/cmd/kube_proxy_replacement.go

```diff
@@ -464,7 +464,16 @@ func detectDevices(detectNodePortDevs, detectDirectRoutingDev bool) error {
 			"Cannot retrieve host IP addrs for BPF NodePort device detection")
 	} else {
 		for _, a := range addrs {
-			if hasHardwareAddress(a.LinkIndex) {
```

Comment:
This line deletion (the `hasHardwareAddress` call) was not part of #16104, but rather part of #14858 - just want to make sure its removal here was intentional. The other changes to this file from #14858 don't appear to be present.

If this was intended, feel free to ignore me. :slightly_smiling_face:

Reply:
It was intended, but thinking it over it might not be a good idea to take it out. Perhaps it would make more sense to keep it included in the `else`.

Reply:
Fixed in latest push.